doc/reduce-memory.md

```diff
@@ -24,9 +24,13 @@ The size of some in-memory caches can be reduced. As caches trade off memory usa
 
 ## Number of peers
 
-- `-maxconnections=<n>` - the maximum number of connections, this defaults to 125. Each active connection takes up some
-  memory. This option applies only if incoming connections are enabled, otherwise the number of connections will never
-  be more than 10. Of the 10 outbound peers, there can be 8 full-relay connections and 2 block-relay-only ones.
+- `-maxconnections=<n>` - the maximum number of connections, which defaults to 125. Each active connection takes up some
+  memory. This option applies only if inbound connections are enabled, otherwise the number of connections will not
```

Comment:
```suggestion
  memory. This option applies only if inbound connections are enabled. Otherwise, the number of connections will not
```
or
```suggestion
  memory. This option applies only if inbound connections are enabled; otherwise, the number of connections will not
```

Reply:
done (thanks!)